Resources/Prototypes/Entities/Structures/Doors/Airlocks/external.yml

```diff
@@ -29,6 +29,17 @@
   id: AirlockExternalGlass
   suffix: Glass, External
   components:
+  - type: Door
+    occludes: false
+    crushDamage:
+      types:
+        Blunt: 15
+    openSound:
+      path: /Audio/Machines/airlock_ext_open.ogg
+    closeSound:
+      path: /Audio/Machines/airlock_ext_close.ogg
+    denySound:
+      path: /Audio/Machines/airlock_deny.ogg
```

Comment:
i dont think you need to re-specify these, they should get inherited

Reply:
Sweet fixed, wasn't sure if it wouldn't inherit since it's the same component overwriting, tested and works fine without it so pushed. Thanks